### PR TITLE
Add support for exporting to WebM

### LINKED ISSUE #86: Webm Support
It would be great if Kap supported exporting video as webm 👍

### PR DESCRIPTION
**Pull request by sindresorhus**
Fixes #86

### CODE REVIEW

File: app/src/renderer/js/main.js [hunk 1]
```diff
@@ -165,15 +165,17 @@ document.addEventListener('DOMContentLoaded', () => {
         windowTitle.innerText = 'Kap';
         time.innerText = '00:00';
         size.innerText = '0 kB';
+        const type = exportAs.value;
 
-        if (exportAs.value === 'mp4') {
+        restoreInputs();
+
+        if (type === 'mp4') {
```
Comment (doot0) [inline app/src/renderer/js/main.js]: Would it be worth changing this to case/switch syntax? IMHO it's bit more readable.

**Comment by matheuss** [comment]
@sindresorhus just did a quick review (3AM right now), and it looks awesome 😌 
Will `checkout` tomorrow to test, but looks great to me 💯

**Comment by sindresorhus** [comment]
@matheuss Should be ready now.

**Comment by matheuss** [comment]
Thank you @sindresorhus ❤️

**Pull request merged**